packages/lesswrong/lib/vendor/ckeditor5-watchdog/contextwatchdog.ts

```diff
@@ -0,0 +1,639 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ *
+ * Vendored from https://github.com/ckeditor/ckeditor5-react
+ * commit 93d7c69b64a5d13dcac79a17794dc278f4d06fb7
+ */
```

Comment:
What's the reason for all this vendoring?

Reply:
We had previously vendored the `ReactCKEditor` component; this corresponds to bringing that vendored component up to date. (It spread into a few more files, between versions.)